modules/caddyhttp/staticresp_test.go

```diff
@@ -44,6 +44,7 @@ func TestStaticResponseHandler(t *testing.T) {
 	}
 
 	resp := w.Result()
+	resp.Body.Close()
 	respBody, _ := io.ReadAll(resp.Body)
```

Comment:
```suggestion
	resp := w.Result()
	respBody, _ := io.ReadAll(resp.Body)
	resp.Body.Close()
```

You want to read the `Body` before closing it ;)